codespell_lib/data/dictionary.txt

```diff
@@ -41588,9 +41716,11 @@ swcloumns->swcolumns
 swepth->swept
 swich->switch
 swiched->switched
+swiches->switches
```

Comment:
Perhaps `swishes`? Probably far-fetched :smile:

Reply:
This typo exists, but is rare. I will add this anyways.